test/benchmark/config/gatewayclass.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
   name: proxy-config
   namespace: envoy-gateway-system
 spec:
+  ipFamily: $ENVOY_PROXY_IP_FAMILY
```

Comment:
neat idea :)

Reply:
the bad news is that we need move all EnvoyProxy resources to this file.